pyrobosim/examples/demo.py

```diff
@@ -97,13 +98,24 @@ def create_world(multirobot=False):
         width_clearance=0.01,
         depth_clearance=0.01,
     )
-    robot = Robot(
-        name="robot",
+
+    robot0 = Robot(
+        name="robot0",
```

Comment:
I guess I forgot to specify, but if you don't give the robot a name, the `World.add_robot()` method should automatically rename them in this same fashion: `robot0`, `robot1`, etc... 

Up to you if you wanna keep these lines for the 3 robots here, no big difference.

Reply:
I was aware of the automatic naming, I just chose to name them explicitly.